docker/resource_docker_container_test.go

```diff
@@ -1580,6 +1586,24 @@ resource "docker_container" "foo" {
 }
 `
 
+const testAccDockerContainerConfig2 = `
```

Comment:
A more speaking name would be appropriate, e.g. `testAccDockerContainerUpdateConfig`

Reply:
Fixed. https://github.com/terraform-providers/terraform-provider-docker/pull/236/commits/0f77c1310ac03a5750930a039a41b43e54690a13